CMakeLists.txt

```diff
@@ -52,6 +52,20 @@ if (NOT BUILD_NUMBER)
   set(BUILD_NUMBER "0")
 endif ()
 
+if (WIN32)
+  # Building enclaves on windows is off by default
+  option(BUILD_ENCLAVES "Build ELF enclaves" OFF)
+else()
+  set(BUILD_ENCLAVES ON)
+endif()
+
+if (BUILD_ENCLAVES AND WIN32)
+  # Setup cross compiler (clangw)
+  add_subdirectory(windows/clangw)
+  include(maybe_build_using_clangw)
```

Comment:
The only thing is that we need to _always_ include this, because it is always used (just a no-op). So on Windows, with `BUILD_ENCLAVES=OFF`, CMake will fail right now.